src/ida_lunde_naalsund_ex/ex04/myrand.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Ida Lunde Naalsund"
+__email__ = "idna@nmbu.no"
+
+
+class LCGRand:
+    """
+    Returns random numbers by implementing the linear congruential
+    generator function.
+
+    Attributes
+    ----------
+    a : int
+        Constant given
+
+    m : int
+        Constant given
+
+    """
+    def __init__(self, seed):
+        """
+
+        Parameters
+        ----------
+        seed: Random seed determined by user
+
+        """
+        self.a = 7**5
+        self.m = 2**31-1
+        self.seed = seed
```

Comment:
i suggest creating a `self.random_number = seed` and using that subsequently. You may store the seed as well, but your design makes the rand() method a bit hard to understand